web/views.py

```
@@ -5181,51 +5181,71 @@ def educational_videos_list(request):
     return render(request, "videos/list.html", context)
 
 
-@login_required
+def fetch_video_oembed(video_url):
```

Ruff (0.8.2)

Missing return type annotation for public function `fetch_video_oembed`

(ANN201)